src/main/java/com/zenith/Proxy.java

```diff
@@ -156,6 +156,35 @@ public void start() {
             // ensure we are continuously updating the tablist even on servers that don't frequently send updates
             SCHEDULED_EXECUTOR_SERVICE.scheduleAtFixedRate(this::tablistUpdate, 20L, 3L, TimeUnit.SECONDS);
             SCHEDULED_EXECUTOR_SERVICE.submit(this::updatePrioBanStatus);
+            // time warning
+            SCHEDULED_EXECUTOR_SERVICE.scheduleAtFixedRate(() -> {
+                try {
```

Comment:
add a check for if the account has prio or not in the runnable:

```
if (this.isPrio.orElse(false)) return; // skip the rest of the method and don't send any warnings
```

there's no 6hr kick for prio accounts